doc/reference/reference_lua/box_schema/func_create.rst

```diff
@@ -6,37 +6,55 @@ box.schema.func.create()
 
 .. module:: box.schema
 
+.. _box_schema-func_create_without-body:
+
 .. function:: box.schema.func.create(func-name [, {options-without-body}])
 
-    Create a function :ref:`tuple <index-box_tuple>`.
+    Create a function :ref:`tuple <index-box_tuple>`
     without including the ``body`` option.
-    (For functions created with the ``body`` option, see
+    For functions created with the ``body`` option, see
     :ref:`box.schema.func.create(func-name [, {options-with-body}]) <box_schema-func_create_with-body>`.
 
     This is called a "not persistent" function because functions without bodies are not persistent.
     This does not create the function itself -- that is done with Lua --
     but if it is necessary to grant privileges for a function,
-    box.schema.func.create must be done first.
+    ``box.schema.func.create`` must be done first.
     For explanation of how Tarantool maintains function data, see the
     reference for the :ref:`box.space._func <box_space-func>` space.
 
     The possible options are:
 
-    * ``if_not_exists`` = ``true|false`` (default = ``false``) - boolean;
-      ``true`` means there should be no error if the ``_func`` tuple already exists.
+    *   ``if_not_exists`` = ``true|false`` (default = ``false``)---
+        ``true`` means there should be no error if the ``_func`` tuple already exists.
+
+    *   ``setuid`` = ``true|false`` (default = ``false``)---with ``true`` to make Tarantool
+        treat the function's caller as the function's creator, with full privileges.
+        Remember that SETUID works only over
+        :ref:`binary ports <admin-security>`.
+        SETUID doesn't work if you invoke a function via an
+        :ref:`admin console <admin-security>` or inside a Lua script.
 
-    * ``setuid`` = ``true|false`` (default = false) - with ``true`` to make Tarantool
-      treat the function’s caller as the function’s creator, with full privileges.
-      Remember that SETUID works only over
-      :ref:`binary ports <admin-security>`.
-      SETUID doesn't work if you invoke a function via an
-      :ref:`admin console <admin-security>` or inside a Lua script.
+    *   ``language`` = 'LUA'|'C' (default = 'LUA').
 
-    * ``language`` = 'LUA'|'C' (default = ‘LUA’).
+    *   ``takes_raw_args`` = ``true|false`` (default = ``false``)---
```

Comment:
Maybe worth adding the version this option first appeared in (2.10.0 that is).

Reply:
Added